src/ImmutableObjectGraph.CodeGeneration.Tests/TestSources/Sealed.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ImmutableObjectGraph.CodeGeneration.Tests.TestSources
+{
+    [GenerateImmutable( DefineInterface = true, DefineWithMethodsPerProperty = true, GenerateBuilder = true )]
```

Comment:
This is the only file with non-VS default whitespace left after your first set of whitespace fixes.